src/web/views/FlowBuilder/common/action.jsx

```diff
@@ -13,23 +13,22 @@ class ActionItem extends Component {
   constructor(props) {
     super(props)
 
-    this.state = { itemId: this.textToItemId(this.props.text) }
-    this.fetchItem(this.props.text)
+    const itemId = this.textToItemId(this.props.text)
+    this.state = { itemId }
+    this.fetchItem(itemId)
   }
 
   textToItemId = text => _.get(text.match(/^say #!(.*)$/), '[1]')
 
-  fetchItem = text => {
-    const itemId = this.textToItemId(text)
-    this.setState({ itemId })
+  fetchItem = itemId => {
     if (itemId) {
-      this.props.fetchContentItem(itemId).then(this.props.refreshFlowsLinks)
+      this.props.fetchContentItem(itemId).then(() => this.props.refreshFlowsLinks())
```

Comment:
Why is this change needed? `refreshFlowsLinks` is unbound anyway

Reply:
Not sure. Possibly it was left after some debugging.